### ISSUE TITLE
[Frontend]Specific imported PersistentVolume imported can't be deleted

### ISSUE THREAD

**Issue by 196Ikuchil** [kubernetes-sigs/kube-scheduler-simulator#160]
## Problem
Specific imported PersistentVolume imported can't be deleted from web UI.
Delete request looks like successful.
Am I mistaken?

Note: 
- That PV has been created by minikube.
- PVs created by me can be deleted.

## Request & Response details

```
DELETE /api/v1/persistentvolumes/pvc-7aa5f39d-7c82-43ce-b10c-50a46c189255 HTTP/1.1
Host: localhost:3131
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:100.0) Gecko/20100101 Firefox/100.0
Accept: application/json, text/plain, */*
Accept-Language: ja,en-US;q=0.7,en;q=0.3
Accept-Encoding: gzip, deflate
Origin: http://localhost:3000
Connection: close
Referer: http://localhost:3000/
Sec-Fetch-Dest: empty
Sec-Fetch-Mode: cors
Sec-Fetch-Site: same-site


HTTP/1.1 200 OK
Access-Control-Allow-Credentials: true
Access-Control-Allow-Headers: Content-Type, Content-Length, Accept-Encoding, X-CSRF-Token, Authorization, X-Requested-With, If-Modified-Since
Access-Control-Allow-Methods: POST, GET, OPTIONS, PUT, DELETE, PATCH
Access-Control-Allow-Origin: http://localhost:3000
Access-Control-Expose-Headers: Date
Audit-Id: 4b94505c-b7b3-438d-9693-e94ab21558d9
Cache-Control: no-cache, private
Content-Type: application/json
X-Kubernetes-Pf-Flowschema-Uid: f7eb3722-d2d3-4e11-8ff9-09c605c587c6
X-Kubernetes-Pf-Prioritylevel-Uid: c82097ed-e6af-428a-b598-048cc77960a4
Date: Thu, 05 May 2022 18:03:13 GMT
Connection: close
Content-Length: 2125

{
  "kind": "PersistentVolume",
  "apiVersion": "v1",
  "metadata": {
    "name": "pvc-7aa5f39d-7c82-43ce-b10c-50a46c189255",
    "uid": "02f1ac3c-02d2-4835-afa6-427e96bd0076",
    "resourceVersion": "242",
    "creationTimestamp": "2022-05-05T17:37:36Z",
    "deletionTimestamp": "2022-05-05T17:38:35Z",
    "deletionGracePeriodSeconds": 0,
    "annotations": {
      "hostPathProvisionerIdentity": "12eb3509-3b1d-45ef-93aa-25fa8b4c7c13",
      "pv.kubernetes.io/provisioned-by": "k8s.io/minikube-hostpath"
    },
    "finalizers": [
      "kubernetes.io/pv-protection"
    ],
    "managedFields": [
      {
        "manager": "simulator",
        "operation": "Apply",
        "apiVersion": "v1",
        "time": "2022-05-05T17:37:36Z",
        "fieldsType": "FieldsV1",
        "fieldsV1": {"f:metadata":{"f:annotations":{"f:hostPathProvisionerIdentity":{},"f:pv.kubernetes.io/provisioned-by":{}},"f:finalizers":{"v:\"kubernetes.io/pv-protection\"":{}}},"f:spec":{"f:accessModes":{},"f:capacity":{"f:storage":{}},"f:claimRef":{},"f:hostPath":{"f:path":{},"f:type":{}},"f:persistentVolumeReclaimPolicy":{},"f:storageClassName":{},"f:volumeMode":{}}}
      },
      {
        "manager": "simulator",
        "operation": "Update",
        "apiVersion": "v1",
        "time": "2022-05-05T17:37:36Z",
        "fieldsType": "FieldsV1",
        "fieldsV1": {"f:status":{"f:phase":{}}},
        "subresource": "status"
      }
    ]
  },
  "spec": {
    "capacity": {
      "storage": "100Mi"
    },
    "hostPath": {
      "path": "/tmp/hostpath-provisioner/kube-scheduler-simulator/kube-scheduler-simulator-etcd-data",
      "type": ""
    },
    "accessModes": [
      "ReadWriteOnce"
    ],
    "claimRef": {
      "kind": "PersistentVolumeClaim",
      "namespace": "kube-scheduler-simulator",
      "name": "kube-scheduler-simulator-etcd-data",
      "uid": "addcaf0a-dbf4-4447-b6c3-99ead48ef6a4",
      "apiVersion": "v1",
      "resourceVersion": "47525"
    },
    "persistentVolumeReclaimPolicy": "Delete",
    "storageClassName": "standard",
    "volumeMode": "Filesystem"
  },
  "status": {
    "phase": "Bound"
  }
}
```

## PV details
```
kind: PersistentVolume
apiVersion: v1
metadata:
  name: pvc-7aa5f39d-7c82-43ce-b10c-50a46c189255
  uid: 02f1ac3c-02d2-4835-afa6-427e96bd0076
  resourceVersion: '242'
  creationTimestamp: '2022-05-05T17:37:36Z'
  deletionTimestamp: '2022-05-05T17:38:35Z'
  deletionGracePeriodSeconds: 0
  annotations:
    hostPathProvisionerIdentity: 12eb3509-3b1d-45ef-93aa-25fa8b4c7c13
    pv.kubernetes.io/provisioned-by: k8s.io/minikube-hostpath
  finalizers:
    - kubernetes.io/pv-protection
  managedFields:
    - manager: simulator
      operation: Apply
      apiVersion: v1
      time: '2022-05-05T17:37:36Z'
      fieldsType: FieldsV1
      fieldsV1:
        f:metadata:
          f:annotations:
            f:hostPathProvisionerIdentity: {}
            f:pv.kubernetes.io/provisioned-by: {}
          f:finalizers:
            v:"kubernetes.io/pv-protection": {}
        f:spec:
          f:accessModes: {}
          f:capacity:
            f:storage: {}
          f:claimRef: {}
          f:hostPath:
            f:path: {}
            f:type: {}
          f:persistentVolumeReclaimPolicy: {}
          f:storageClassName: {}
          f:volumeMode: {}
    - manager: simulator
      operation: Update
      apiVersion: v1
      time: '2022-05-05T17:37:36Z'
      fieldsType: FieldsV1
      fieldsV1:
        f:status:
          f:phase: {}
      subresource: status
spec:
  capacity:
    storage: 100Mi
  hostPath:
    path: >-
      /tmp/hostpath-provisioner/kube-scheduler-simulator/kube-scheduler-simulator-etcd-data
    type: ''
  accessModes:
    - ReadWriteOnce
  claimRef:
    kind: PersistentVolumeClaim
    namespace: kube-scheduler-simulator
    name: kube-scheduler-simulator-etcd-data
    uid: addcaf0a-dbf4-4447-b6c3-99ead48ef6a4
    apiVersion: v1
    resourceVersion: '47525'
  persistentVolumeReclaimPolicy: Delete
  storageClassName: standard
  volumeMode: Filesystem
status:
  phase: Bound
```

**Comment by 196Ikuchil** [comment]
## On successful
PVs created by me can be deleted.
```
DELETE /api/v1/persistentvolumes/pv2 HTTP/1.1
Host: localhost:3131
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:100.0) Gecko/20100101 Firefox/100.0
Accept: application/json, text/plain, */*
Accept-Language: ja,en-US;q=0.7,en;q=0.3
Accept-Encoding: gzip, deflate
Origin: http://localhost:3000
Connection: close
Referer: http://localhost:3000/
Sec-Fetch-Dest: empty
Sec-Fetch-Mode: cors
Sec-Fetch-Site: same-site


HTTP/1.1 200 OK
Access-Control-Allow-Credentials: true
Access-Control-Allow-Headers: Content-Type, Content-Length, Accept-Encoding, X-CSRF-Token, Authorization, X-Requested-With, If-Modified-Since
Access-Control-Allow-Methods: POST, GET, OPTIONS, PUT, DELETE, PATCH
Access-Control-Allow-Origin: http://localhost:3000
Access-Control-Expose-Headers: Date
Audit-Id: 1ac7d581-9620-48a1-9443-36a1febb00b5
Cache-Control: no-cache, private
Content-Type: application/json
X-Kubernetes-Pf-Flowschema-Uid: f7eb3722-d2d3-4e11-8ff9-09c605c587c6
X-Kubernetes-Pf-Prioritylevel-Uid: c82097ed-e6af-428a-b598-048cc77960a4
Date: Fri, 06 May 2022 01:47:19 GMT
Content-Length: 1224
Connection: close

{
  "kind": "PersistentVolume",
  "apiVersion": "v1",
  "metadata": {
    "name": "pv2",
    "uid": "ef883531-a924-4d76-bf46-7b3380be751d",
    "resourceVersion": "611",
    "creationTimestamp": "2022-05-06T01:47:13Z",
    "managedFields": [
      {
        "manager": "simulator",
        "operation": "Apply",
        "apiVersion": "v1",
        "time": "2022-05-06T01:47:13Z",
        "fieldsType": "FieldsV1",
        "fieldsV1": {"f:metadata":{"f:labels":{}},"f:spec":{"f:accessModes":{},"f:capacity":{"f:storage":{}},"f:hostPath":{"f:path":{},"f:type":{}},"f:persistentVolumeReclaimPolicy":{},"f:volumeMode":{}}}
      },
      {
        "manager": "simulator",
        "operation": "Update",
        "apiVersion": "v1",
        "time": "2022-05-06T01:47:13Z",
        "fieldsType": "FieldsV1",
        "fieldsV1": {"f:status":{"f:phase":{}}},
        "subresource": "status"
      }
    ]
  },
  "spec": {
    "capacity": {
      "storage": "1Gi"
    },
    "hostPath": {
      "path": "/tmp/data",
      "type": "DirectoryOrCreate"
    },
    "accessModes": [
      "ReadWriteOnce"
    ],
    "persistentVolumeReclaimPolicy": "Delete",
    "volumeMode": "Filesystem"
  },
  "status": {
    "phase": "Available"
  }
}
```

**Comment by 196Ikuchil** [comment]
I found the cause.
Because we can't delete the other namespaced PVC from web UI, we also fail to delete the PV.
```
  claimRef:
    kind: PersistentVolumeClaim
    namespace: kube-scheduler-simulator
    name: kube-scheduler-simulator-etcd-data
    uid: addcaf0a-dbf4-4447-b6c3-99ead48ef6a4
    apiVersion: v1
    resourceVersion: '47525'
```
This issue may be solved by #84. So, this is a duplication.
/close
